rust/main/src/cli/commands/node/generate_config.rs

```diff
@@ -25,21 +21,14 @@ pub(crate) struct GenerateConfigArgs {
 
 impl GenerateConfigArgs {
     pub(crate) fn generate_config(&self) -> Result<()> {
-        let mut toml = TomlConfig::new();
-        let network = NetworkConstants::active_network();
-        let mut config_type = "node";
-
-        if self.node {
-            let network_params = NetworkParams::new(network);
-            let mut config = DaemonConfig::new(&network_params, 0);
-            config.node.peering_port = Some(network_params.network.default_node_port);
-            config.serialize_toml(&mut toml)?
+        let (toml_str, config_type) = if self.node {
+            let daemon_toml = DaemonToml::default();
+            (toml::to_string(&daemon_toml)?, "node")
         } else {
-            config_type = "rpc";
-            let network_constants = NetworkConstants::new(WorkThresholds::new(0, 0, 0), network);
-            let config = RpcConfig::new(&network_constants, 0);
-            config.serialize_toml(&mut toml)?
-        }
+            //let rpc_toml = RpcToml::default();
```

Comment:
Why is this commented out? Is RpcToml not ready yet?

Reply:
It is implemented in #27 .